administration/src/cards/extensions/KoblenzReferenceNumberExtension.tsx

```diff
@@ -11,6 +12,9 @@ type KoblenzReferenceNumberExtensionState = { [KOBLENZ_REFERENCE_NUMBER_EXTENSIO
 
 const KoblenzReferenceNumberMinLength = 4
 const KoblenzReferenceNumberMaxLength = 15
+const hasSpecialChars = (referenceNr: string): boolean => /[`!@#$%^&*()_+\-=\]{};':"\\|,<>?~]/.test(referenceNr)
```

Comment:
:thinking: Is this a different regex compared to e.g. the name? If yes, what are the differences? Why?

Reply:
yes because "." is allowed here

Reply:
Same as above, i would restrict this to latin-letters (or all letters) and numbers and dot unicode ranges

Reply:
i don't think we have to restrict that much here, because it has to fit to the hash and if it doesn't you can not create a pdf and it won't break

Reply:
But you probably its general a good idea to add this check to all form fields. But then we also may to show a proper error message. I kept this issue open https://github.com/digitalfabrik/entitlementcard/issues/1776
There we should check all form fields and ensure that even card creation (nuernberg & bayern) can not break due to wrong charset